src/main/asciidoc/java/parsetools.adoc

```diff
@@ -47,4 +47,164 @@ RecordParser.newFixed(4, h -> {
 });
 ----
 
-For more details, check out the `link:../../apidocs/io/vertx/core/parsetools/RecordParser.html[RecordParser]` class.
\ No newline at end of file
+For more details, check out the `link:../../apidocs/io/vertx/core/parsetools/RecordParser.html[RecordParser]` class.
+
+== Json Parser
+
+You can easily parse JSON structures but that requires to provide the JSON string at once, but it
```

Comment:
Minor: _JSON content_ instead of  _JSON string_. It's not always a string, it can be a buffer as well

Reply:
the buffer is encoded as a JSON string though

Reply:
ah ok, I see what you mean :-)